sysinfo.go

```diff
@@ -32,13 +35,51 @@ const (
 	ProcessorArchitectureAMD64   ProcessorArchitecture = wrappers.PROCESSOR_ARCHITECTURE_AMD64
 )
 
+type DisplayDevice struct {
+	DeviceName   string
+	DeviceString string
+	StateFlags   uint32
+	DeviceID     string
+	DeviceKey    string
+}
+
+type DisplayMonitorInfo struct {
+	Handle wrappers.HMONITOR
+	Hdc    wrappers.HDC
+	Rect   wrappers.RECT
```

Comment:
Let's have a structure on the Go side to wrap `RECT`.  The natural name for it would be `Rectangle`.  We could have a file called geometry.go to hold it.  If we add structures wrapping `POINT` and `SIZE` at some point in the future, they can go in that file as well.

Reply:
I defined Rectangle with int fields. I'm not sure  - maybe it should be int32 ... 
I also added rectToRectangle helper.